share/scripts/inverse/csg_table

```diff
@@ -96,6 +96,10 @@ postadd overwrite postadd_overwrite.sh
 postadd plot postadd_plot.sh
 postadd average postadd_average.sh
 
+# smoothing for updated potential to allow forgetting noise from previous iterations (rdfs)
+# the script is the same as postupd_smooth.sh
+postadd smooth postadd_smooth.sh
```

Comment:
Smoothing of the potential after adding (`postadd`) will lead to washing out of features, which is we don't have it, see https://groups.google.com/d/topic/votca/h2Lqi62Ncrs/discussion.

So, why does the normal post-update smoothing don't work for you?